tests/zfs-tests/cmd/xattrtest/xattrtest.c

```diff
@@ -44,7 +44,11 @@
 #include <sys/time.h>
 #include <linux/limits.h>
 
+#ifdef __UCLIBC__
+extern const char *program_invocation_short_name;
+#else
 extern char *program_invocation_short_name;
+#endif
```

Comment:
For what it's worth, for this test program I think it'd be fine if we just hardcoded the name and removed the need for this.  It's never called by another name, and this is the only place in the code base which uses `program_invocation_short_name`.  If you want to make that code change I'd be all for it.

Reply:
@behlendorf I just published another patch with your suggestion. Thanks.